op-node/flags/flags.go

```diff
@@ -73,6 +73,13 @@ var (
 		EnvVars:  prefixEnvVars("L1_BEACON"),
 		Category: RollupCategory,
 	}
+	SupervisorAddr = &cli.StringFlag{
+		Name: "supervisor",
+		Usage: "RPC address of interop supervisor service for cross-chain safety verification." +
+			"Applies only to Interop-enabled networks.",
+		Hidden:  true, // hidden for now during early testing.
```

Comment:
Good idea, but can we make sure we have an issue somewhere to unhide this - it's easy to forget. :)

Reply:
Added: https://github.com/ethereum-optimism/optimism/issues/11774